document/src/tests/documentupdatetestcase.cpp

```diff
@@ -697,18 +594,17 @@ DocumentUpdateTest::testUpdateApplyNoArrayValues()
 
     // Verify that the field was set in the document
     std::unique_ptr<ArrayFieldValue> fval(doc->getAs<ArrayFieldValue>(field));
-    CPPUNIT_ASSERT(fval.get());
-    CPPUNIT_ASSERT_EQUAL((size_t) 0, fval->size());
+    EXPECT_TRUE(fval.get());
```

Comment:
Consider `ASSERT_TRUE`, is dereferenced in the following expectation

Reply:
Fixed